tools/build.mjs

```diff
@@ -1,38 +1,42 @@
-import * as fs from "fs/promises";
-import { rimraf } from "rimraf";
-import cp from "child_process";
-import zipAFolder from "zip-a-folder";
+import * as fs from 'fs/promises'
+import { rimraf } from 'rimraf'
+import cp from 'child_process'
+import zipAFolder from 'zip-a-folder'
+import mkdirp from 'mkdirp'
 
-const platform = process.argv[2] || process.platform;
-const arch = process.argv[3] || process.arch;
+const platform = process.argv[2] || process.platform
+const arch = process.argv[3] || process.arch
 
-console.log(`Building for ${platform}-${arch}`);
+console.log(`Building for ${platform}-${arch}`)
 
-await rimraf("dist");
+await rimraf('deploy')
+mkdirp.sync('deploy')
 
 // Run pkg
-const filename = platform === "win32" ? "dist/scanner.exe" : "dist/scanner";
-cp.execSync(`pkg -t node18-${platform} src/index.js -o ${filename}`);
+const filename = platform === 'win32' ? 'deploy/scanner.exe' : 'deploy/scanner'
+try {
+	cp.execSync(`pkg -t node18-${platform} dist/index.js -o ${filename}`)
+} catch (error) {
+	console.log(error.stdout.toString())
```

Comment:
this should do an `process.exit(1)`, or rethrow the error so that the script terminates when this fails